commands/files/serialfile.go

```diff
@@ -84,10 +84,25 @@ func (f *serialFile) NextFile() (File, error) {
 	// open the next file
 	fileName := fp.Join(f.name, stat.Name())
 	filePath := fp.Join(f.path, stat.Name())
+	st, err := os.Lstat(filePath)
```

Comment:
The option should not be a hard-coded "only create symlink if '-r'", but rather another flag to dereference the symlinks, regardless it is a single symlink or '-r' symlink.

If it is potentially dangerous to create a single symlink, then there should be a warning message instead, not hard-coded.